learning/tour-of-beam/frontend/lib/components/footer.dart

```diff
@@ -31,12 +30,25 @@ class Footer extends StatelessWidget {
   Widget build(BuildContext context) {
     return _Body(
       child: Wrap(
-        spacing: TobSizes.size16,
+        alignment: WrapAlignment.spaceBetween,
         crossAxisAlignment: WrapCrossAlignment.center,
         children: [
-          const _ReportIssueButton(),
-          const _PrivacyPolicyButton(),
-          const Text('ui.copyright').tr(),
+          Wrap(
+            spacing: BeamSizes.size16,
+            crossAxisAlignment: WrapCrossAlignment.center,
+            children: [
+              const _ReportIssueButton(),
+              const _PrivacyPolicyButton(),
+              const Text('ui.copyright').tr(),
+            ],
+          ),
+          // TODO(nausharipov): get version
```

Comment:
This deserves to be an issue in the tracker linked here. File one if we don't have it yet.

Reply:
Done: https://github.com/apache/beam/issues/23038

Reply:
Please next time add the issue URL to the TODO comment itself. I will add this time.